### DISCUSSION TITLE
Can we use in-cluster prometheus instead of workflow controller's prometheus to report metrics?

### DISCUSSION THREAD

**Discussion by liujiong25** [argoproj/argo-workflows#10179] [General]
I have a prometheus installed in the cluster
Can we use my installed prometheus instead of workflow controller's prometheus to report metrics?

Also how to set alarm for prometheus metrics, like if workflow is more than 50, send alarm to me?

**Comment by sarabala1979** [comment]
The workflow controller doesn't have Prometheus. you can configure your Prometheus to scrape the metrics from the workflow controller.

**Reply by liujiong25** [reply]
is there an example or doc for how to do it?

**Reply by liujiong25** [reply]
I can see the metrics at host:9090/metrics, but how do i let my installed prometheus pull that metrics?
